webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageUser.vue

```diff
@@ -42,44 +90,47 @@ export default {
         'user full name': this.profilelink,
       });
     },
-    companyName() {
-      return this.branding && this.branding.companyName;
+    emptyStreamForSpacesThatUserManage1() {
+      return this.$t('UIActivity.label.empty_space_stream_manage_community');
     },
-    thanksForJoingMeeds() {
-      return this.$t && this.$t('UIActivity.label.Thanks_for_joing_company', {
-        'company name': this.companyName,
-      });
+    emptyStreamForSpacesThatUserManage2() {
+      return this.$t('UIActivity.label.empty_space_stream_share_knowledge');
     },
-    welcomeParagraph1() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_paragraph_one', {
-        'product name': this.productName,
-      });
+    emptyStreamForSpacesThatUserManage3() {
+      return this.$t('UIActivity.label.empty_space_stream_congrats');
     },
-    welcomeParagraph2() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_paragraph_two', {
-        'product name': this.productName,
-      });
+    emptyStreamForSpacesThatUserManage4() {
+      return this.$t('UIActivity.label.empty_space_stream_value_skills');
     },
-    welcomeParagraph3() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_paragraph_three');
+    welcomeSpaceParagraph1() {
+      return this.$t('UIActivity.label.empty_space_stream_paragraph_one');
     },
-    welcomeParagraph4() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_paragraph_four');
+    welcomeSpaceParagraph2() {
+      return this.$t('UIActivity.label.empty_space_stream_paragraph_two');
     },
-    welcomeParagraph5() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_paragraph_five');
+    welcomeSpaceParagraph3() {
+      return this.$t('UIActivity.label.empty_space_stream_paragraph_three');
     },
-    conclusionParagraph() {
-      return this.$t && this.$t('UIActivity.label.empty_stream_lets_begin');
+    welcomeSpaceParagraph4() {
+      return this.$t('UIActivity.label.empty_space_stream_paragraph_four');
+    },
+    welcomeSpaceNumberedParagraph1() {
+      return this.$t('UIActivity.label.empty_space_stream_paragraph_numbered_one', {
+        0: `<a class="primary--text font-weight-bold" href="javascript:void(0)" onclick="document.dispatchEvent(new CustomEvent('${this.emptyActivityStreamActionName}'))">`,
```

Comment:
make an `href` attribute here instead of dealing with events to redirect to a page.

Reply:
done